### DISCUSSION TITLE
Reference for dbs-properties.json

### DISCUSSION THREAD

**Discussion by HughWarrington** [joyfullservice/msaccess-vcs-addin#556] [Q&A] [answered]
Where can I find out what all the contents of this file are about?

Specifically I'm wondering about AppTitle vs Name which have the same value in my case -- not sure what effect changing Name will have.

An answer which just points me to relevant code would be fine too :)

**Comment by joyfullservice** [comment, marked as answer]
If you search the VBA code for `dbs-properties.json`, it will bring you to `clsDbProperty`. Essentially it is building a list of `CurrentDb.Properties` with a few exclusions and adjustments for relative file names.

**Reply by HughWarrington** [reply]
https://github.com/joyfullservice/msaccess-vcs-addin/blob/668f940e93d6ba465024b50f5c982ee4e80793bc/Version%20Control.accda.src/modules/clsDbProperty.cls#L248
